packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -49,6 +52,7 @@ export async function prefetchLatestSnapshot(
     hostSnapshotFetchOptions: ISnapshotOptions | undefined,
     enableRedeemFallback?: boolean,
     fetchBinarySnapshotFormat?: boolean,
```

Comment:
fetchBinarySnapshotFormat arg is not used, it should be deleted, right?

Reply:
I am deprecating it first before removing as it is an external api.